### PR TITLE
Add IBM Plex font

### PR DESCRIPTION
**Pull request by K0ng2**
Adds IBM Plex to the bucket.

- IBM Plex Mono
- IBM Plex Sans
- IBM Plex Sans Arabic
- IBM Plex Sans Condensed
- IBM Plex Sans Devanagari
- IBM Plex Sans Hebrew
- IBM Plex Sans JP
- IBM Plex Sans KR
- IBM Plex Sans Thai
- IBM Plex Sans Thai Looped
- IBM Plex Serif

Github source repo:
- [IBM Plex](https://github.com/IBM/plex)

### CODE REVIEW

File: bucket/IBMPlexMono.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "version": "6.0.0",
+  "license": "SIL OFL 1.1",
```
Comment (issaclin32) [inline bucket/IBMPlexMono.json:3]: Please use `OFL-1.1` instead.
See https://spdx.org/licenses/

**Review by issaclin32 — APPROVED** [review]

File: bucket/IBMPlexSans-JP.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "version": "6.0.0",
+  "license": "OFL-1.1",
+  "homepage": "https://github.com/IBM/plex",
+  "url": "https://github.com/IBM/plex/releases/download/v6.0.0/TrueType.zip",
+  "hash": "16253500de140321175bb94e65fd227c6b12efc050eb298c20322dbf3713a8e0",
+  "extract_dir": "TrueType/IBM-Plex-Sans-JP",
```
Comment (jonz94) [inline bucket/IBMPlexSans-JP.json:7]:
```suggestion
  "extract_dir": "TrueType/IBM-Plex-Sans-JP/hinted",
```

File: bucket/IBMPlexSans-KR.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "version": "6.0.0",
+  "license": "OFL-1.1",
+  "homepage": "https://github.com/IBM/plex",
+  "url": "https://github.com/IBM/plex/releases/download/v6.0.0/TrueType.zip",
+  "hash": "16253500de140321175bb94e65fd227c6b12efc050eb298c20322dbf3713a8e0",
+  "extract_dir": "TrueType/IBM-Plex-Sans-KR",
```
Comment (jonz94) [inline bucket/IBMPlexSans-KR.json:7]:
```suggestion
  "extract_dir": "TrueType/IBM-Plex-Sans-KR/hinted",
```

**Review by jonz94 — CHANGES_REQUESTED** [review]

File: bucket/IBMPlexSans-JP.json (hunk 2 above)
Comment (jonz94) [reply bucket/IBMPlexSans-JP.json:7]: <details>
<summary>structure of the zip file</summary>

```
$ tree TrueType
TrueType
├── IBM-Plex-Mono
│   ├── IBMPlexMono-BoldItalic.ttf
│   ├── IBMPlexMono-Bold.ttf
│   ├── IBMPlexMono-ExtraLightItalic.ttf
│   ├── IBMPlexMono-ExtraLight.ttf
│   ├── IBMPlexMono-Italic.ttf
│   ├── IBMPlexMono-LightItalic.ttf
│   ├── IBMPlexMono-Light.ttf
│   ├── IBMPlexMono-MediumItalic.ttf
│   ├── IBMPlexMono-Medium.ttf
│   ├── IBMPlexMono-Regular.ttf
│   ├── IBMPlexMono-SemiBoldItalic.ttf
│   ├── IBMPlexMono-SemiBold.ttf
│   ├── IBMPlexMono-TextItalic.ttf
│   ├── IBMPlexMono-Text.ttf
│   ├── IBMPlexMono-ThinItalic.ttf
│   ├── IBMPlexMono-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans
│   ├── IBMPlexSans-BoldItalic.ttf
│   ├── IBMPlexSans-Bold.ttf
│   ├── IBMPlexSans-ExtraLightItalic.ttf
│   ├── IBMPlexSans-ExtraLight.ttf
│   ├── IBMPlexSans-Italic.ttf
│   ├── IBMPlexSans-LightItalic.ttf
│   ├── IBMPlexSans-Light.ttf
│   ├── IBMPlexSans-MediumItalic.ttf
│   ├── IBMPlexSans-Medium.ttf
│   ├── IBMPlexSans-Regular.ttf
│   ├── IBMPlexSans-SemiBoldItalic.ttf
│   ├── IBMPlexSans-SemiBold.ttf
│   ├── IBMPlexSans-TextItalic.ttf
│   ├── IBMPlexSans-Text.ttf
│   ├── IBMPlexSans-ThinItalic.ttf
│   ├── IBMPlexSans-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-Arabic
│   ├── IBMPlexSansArabic-Bold.ttf
│   ├── IBMPlexSansArabic-ExtraLight.ttf
│   ├── IBMPlexSansArabic-Light.ttf
│   ├── IBMPlexSansArabic-Medium.ttf
│   ├── IBMPlexSansArabic-Regular.ttf
│   ├── IBMPlexSansArabic-SemiBold.ttf
│   ├── IBMPlexSansArabic-Text.ttf
│   ├── IBMPlexSansArabic-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-Condensed
│   ├── IBMPlexSansCondensed-BoldItalic.ttf
│   ├── IBMPlexSansCondensed-Bold.ttf
│   ├── IBMPlexSansCondensed-ExtraLightItalic.ttf
│   ├── IBMPlexSansCondensed-ExtraLight.ttf
│   ├── IBMPlexSansCondensed-Italic.ttf
│   ├── IBMPlexSansCondensed-LightItalic.ttf
│   ├── IBMPlexSansCondensed-Light.ttf
│   ├── IBMPlexSansCondensed-MediumItalic.ttf
│   ├── IBMPlexSansCondensed-Medium.ttf
│   ├── IBMPlexSansCondensed-Regular.ttf
│   ├── IBMPlexSansCondensed-SemiBoldItalic.ttf
│   ├── IBMPlexSansCondensed-SemiBold.ttf
│   ├── IBMPlexSansCondensed-TextItalic.ttf
│   ├── IBMPlexSansCondensed-Text.ttf
│   ├── IBMPlexSansCondensed-ThinItalic.ttf
│   ├── IBMPlexSansCondensed-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-Devanagari
│   ├── IBMPlexSansDevanagari-Bold.ttf
│   ├── IBMPlexSansDevanagari-ExtraLight.ttf
│   ├── IBMPlexSansDevanagari-Light.ttf
│   ├── IBMPlexSansDevanagari-Medium.ttf
│   ├── IBMPlexSansDevanagari-Regular.ttf
│   ├── IBMPlexSansDevanagari-SemiBold.ttf
│   ├── IBMPlexSansDevanagari-Text.ttf
│   ├── IBMPlexSansDevanagari-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-Hebrew
│   ├── IBMPlexSansHebrew-Bold.ttf
│   ├── IBMPlexSansHebrew-ExtraLight.ttf
│   ├── IBMPlexSansHebrew-Light.ttf
│   ├── IBMPlexSansHebrew-Medium.ttf
│   ├── IBMPlexSansHebrew-Regular.ttf
│   ├── IBMPlexSansHebrew-SemiBold.ttf
│   ├── IBMPlexSansHebrew-Text.ttf
│   ├── IBMPlexSansHebrew-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-JP
│   ├── hinted
│   │   ├── IBMPlexSansJP-Bold.ttf
│   │   ├── IBMPlexSansJP-ExtraLight.ttf
│   │   ├── IBMPlexSansJP-Light.ttf
│   │   ├── IBMPlexSansJP-Medium.ttf
│   │   ├── IBMPlexSansJP-Regular.ttf
│   │   ├── IBMPlexSansJP-SemiBold.ttf
│   │   ├── IBMPlexSansJP-Text.ttf
│   │   ├── IBMPlexSansJP-Thin.ttf
│   │   └── license.txt
│   └── unhinted
│       ├── IBMPlexSansJP-Bold.ttf
│       ├── IBMPlexSansJP-ExtraLight.ttf
│       ├── IBMPlexSansJP-Light.ttf
│       ├── IBMPlexSansJP-Medium.ttf
│       ├── IBMPlexSansJP-Regular.ttf
│       ├── IBMPlexSansJP-SemiBold.ttf
│       ├── IBMPlexSansJP-Text.ttf
│       ├── IBMPlexSansJP-Thin.ttf
│       └── license.txt
├── IBM-Plex-Sans-KR
│   ├── hinted
│   │   ├── IBMPlexSansKR-Bold.ttf
│   │   ├── IBMPlexSansKR-ExtraLight.ttf
│   │   ├── IBMPlexSansKR-Light.ttf
│   │   ├── IBMPlexSansKR-Medium.ttf
│   │   ├── IBMPlexSansKR-Regular.ttf
│   │   ├── IBMPlexSansKR-SemiBold.ttf
│   │   ├── IBMPlexSansKR-Text.ttf
│   │   ├── IBMPlexSansKR-Thin.ttf
│   │   └── license.txt
│   └── unhinted
│       ├── IBMPlexSansKR-Bold.ttf
│       ├── IBMPlexSansKR-ExtraLight.ttf
│       ├── IBMPlexSansKR-Light.ttf
│       ├── IBMPlexSansKR-Medium.ttf
│       ├── IBMPlexSansKR-Regular.ttf
│       ├── IBMPlexSansKR-SemiBold.ttf
│       ├── IBMPlexSansKR-Text.ttf
│       ├── IBMPlexSansKR-Thin.ttf
│       └── license.txt
├── IBM-Plex-Sans-Thai
│   ├── IBMPlexSansThai-Bold.ttf
│   ├── IBMPlexSansThai-ExtraLight.ttf
│   ├── IBMPlexSansThai-Light.ttf
│   ├── IBMPlexSansThai-Medium.ttf
│   ├── IBMPlexSansThai-Regular.ttf
│   ├── IBMPlexSansThai-SemiBold.ttf
│   ├── IBMPlexSansThai-Text.ttf
│   ├── IBMPlexSansThai-Thin.ttf
│   └── license.txt
├── IBM-Plex-Sans-Thai-Looped
│   ├── IBMPlexSansThaiLooped-Bold.ttf
│   ├── IBMPlexSansThaiLooped-ExtraLight.ttf
│   ├── IBMPlexSansThaiLooped-Light.ttf
│   ├── IBMPlexSansThaiLooped-Medium.ttf
│   ├── IBMPlexSansThaiLooped-Regular.ttf
│   ├── IBMPlexSansThaiLooped-SemiBold.ttf
│   ├── IBMPlexSansThaiLooped-Text.ttf
│   ├── IBMPlexSansThaiLooped-Thin.ttf
│   └── license.txt
└── IBM-Plex-Serif
    ├── IBMPlexSerif-BoldItalic.ttf
    ├── IBMPlexSerif-Bold.ttf
    ├── IBMPlexSerif-ExtraLightItalic.ttf
    ├── IBMPlexSerif-ExtraLight.ttf
    ├── IBMPlexSerif-Italic.ttf
    ├── IBMPlexSerif-LightItalic.ttf
    ├── IBMPlexSerif-Light.ttf
    ├── IBMPlexSerif-MediumItalic.ttf
    ├── IBMPlexSerif-Medium.ttf
    ├── IBMPlexSerif-Regular.ttf
    ├── IBMPlexSerif-SemiBoldItalic.ttf
    ├── IBMPlexSerif-SemiBold.ttf
    ├── IBMPlexSerif-TextItalic.ttf
    ├── IBMPlexSerif-Text.ttf
    ├── IBMPlexSerif-ThinItalic.ttf
    ├── IBMPlexSerif-Thin.ttf
    └── license.txt

15 directories, 149 files
```

</details>

I found that `JP` and `KR` fonts have different folder structure.
Just need some minor adjustments, and everything else looks good to me 👍

**Review by jonz94 — APPROVED** [review]

**Pull request merged**